--- origsrc/libxml2-2.9.2/configure.ac	2014-10-15 23:06:15.000000000 -0500
+++ src/libxml2-2.9.2/configure.ac	2014-11-16 17:38:34.340728900 -0600
@@ -392,7 +392,9 @@ if test "$with_zlib" = "no"; then
 else
     AC_CHECK_HEADERS(zlib.h,
         [SAVE_LDFLAGS="${LDFLAGS}"
+	 if test "x${Z_DIR}" != "x"; then
 	 LDFLAGS="-L${Z_DIR}/lib"
+	 fi
 	AC_CHECK_LIB(z, gzread,[
 	    AC_DEFINE([HAVE_LIBZ], [1], [Have compression library])
 	    WITH_ZLIB=1
